frontend/src/redux/reducers/isUserAuthorizedReducer.tsx

```diff
@@ -5,7 +5,7 @@ import {
 import { isUserAuthorizedStateType } from '../ts-types';
 
 const initialState: isUserAuthorizedStateType = {
-  loading: true,
+  loading: false,
```

Comment:
in this place why you changed this field?

Reply:
In my opinion, it is right when the initial loading state is false. I didn't find any issues after these changes. And, finally, I need the false state for the correct work of my task.